api/shape/z.js

```diff
@@ -4,6 +4,6 @@ import move from './move.js';
 export const z =
   (...z) =>
   (shape) =>
-    Shape.Group(...shape.toFlatValues(z).map((z) => move(0, 0, z)(shape)));
+    Shape.Group(...shape.toFlatValues(z).map((z) => move([0, 0, z])(shape)));
```

Comment:
Are we moving away from (x,y,z) and towards ([x,y,z]) being the standard?

Reply:
Not exactly, rather moving toward them both coexisting.

But this form is a bit simpler to process and less ambiguous.